rockcraft.yaml

```diff
@@ -44,8 +45,8 @@ parts:
     source-tag: v7.9.0
 
   deb-security-manifest:
-    plugin: nil
-    override-prime: |
-      set -x
-      mkdir -p $CRAFT_PRIME/usr/share/rocks/
-      (echo "# os-release" && cat /etc/os-release && echo "# dpkg-query" && dpkg-query --admindir=$CRAFT_PRIME/var/lib/dpkg/ -f '${db:Status-Abbrev},${binary:Package},${Version},${source:Package},${Source:Version}\n' -W) > $CRAFT_PRIME/usr/share/rocks/dpkg.query
+    plugin: make
+    source: https://github.com/canonical/rocks-security-manifest
+    source-type: git
+    source-branch: main
```

Comment:
shouldn't we pin to a tag?this feels risky

Reply:
https://github.com/canonical/kratos-rock/pull/203#discussion_r2216141947